README.md

```diff
@@ -239,6 +244,91 @@ services:
         # greater than this value, it is capped to this value. Default is
         # half a minute ("30s").
         backoff-limit: <duration>
+
+# (Optional) A list of health checks managed by this configuration layer.
+checks:
+    <check name>:
+        # (Required) Control how this check definition is combined with any
+        # other pre-existing definition with the same name in the Pebble plan.
+        #
+        # The value 'merge' will ensure that values in this layer specification
+        # are merged over existing definitions, whereas 'replace' will entirely
+        # override the existing check spec in the plan with the same name.
+        override: merge | replace
+
+        # (Optional) Check level, which can be used for filtering checks when
+        # calling the checks API or health endpoint. For the health endpoint,
+        # ready implies alive.
+        level: alive | ready
+
+        # (Optional) Check is run every time this period (time interval)
+        # elapses. Must not be zero. Default is "10s".
+        period: <duration>
+
+        # (Optional) If this time elapses before each check has finished, it
+        # is cancelled and considered to have failed. Must not be less than
+        # the period, and must not be zero. Default is "3s".
+        timeout: <duration>
+
+        # (Optional) Number of times the check must in a row to be considered
+        # failing and to trigger the on-check-failure action. Default 3.
+        failures: <failure threshold>
+
+        # Type-specific check configuration is below. Note that one (and only
```

Comment:
Consider dropping this comment for my suggestion below.

Reply:
Done.